CHANGELOG.md

```diff
@@ -1,3 +1,21 @@
+Bump version to 0.14.0
```

Comment:
Can you make this the same style as other version changes? It's not a heading and has a different phrasing (see https://github.com/alphagov/smartdown/blob/v-0-14-0/CHANGELOG.md)

You just need 
```## 0.14.0```

Otherwise, LGTM :+1: